mod/node-core/pkg/components/depinject.go

```diff
@@ -62,7 +62,7 @@ type coreKVStore struct {
 
 // NewKVStore returns a wrapper of Core/Store kvstore interface
 // Remove once store migrates to core/store kvstore interface.
-func newKVStore(store storetypes.KVStore) store.KVStore {
+func NewKVStore(store storetypes.KVStore) store.KVStore {
```

Comment:
I need this exported to duly initialize a store in unit tests

Reply:
I am wondering if it is a good practice to export a function, just for UTs.

Reply:
Not great, but I think the worst is not having proper test coverage.
My bet with exporting this function is that other packages' unit tests will need this function to instantiate their in-memory DB and KVStore.
If I am wrong, we can always open a new PR, un export this function and structure `storage` unit tests differently.
What do you think?